catalog/spatial/wfs/2.0.0/spatial-wfs-v2_0_0-source/src/test/java/org/codice/ddf/spatial/ogc/wfs/v2_0_0/catalog/source/TestWfsFilterDelegate.java

```diff
@@ -138,12 +144,44 @@
 
   private String mockFeatureType = "mockFeatureType";
 
-  private String mockFeatureProperty = "mockFeatureProperty";
+  private String mockFeatureProperty = "localpart.EXACT_COLLECT_DATE";
 
   private MetacardMapper mockMapper;
 
   private FeatureMetacardType mockFeatureMetacardType = mock(FeatureMetacardType.class);
 
+  private List<Entry> mappingEntries;
+
+  @Before
+  public void setup() {
+    Serializable[][] mapping = {{mockMetacardAttribute, mockFeatureProperty, new Date()}};
+    this.mappingEntries = new ArrayList<>();
+    this.mockMapper = mock(MetacardMapper.class);
+
+    Stream.of(mapping).forEach(this::addMockMetacardMapperEntry);
+
+    when(mockMapper.stream()).thenAnswer((i) -> mappingEntries.stream());
+
+    when(mockMapper.getEntry(any(Predicate.class)))
+        .thenAnswer(
+            (i) -> {
+              Predicate<Entry> p = (Predicate<Entry>) i.getArguments()[0];
+              return mappingEntries.stream().filter(p).findFirst();
+            });
+  }
+
+  private void addMockMetacardMapperEntry(Serializable[] mapping) {
```

Comment:
This method seems to repeated for WFS 1.0, WFS 1.1 and WFS 2.0. Could it be refactored into a common class?